homeassistant/components/atag/climate.py

```diff
@@ -49,21 +53,14 @@ def hvac_mode(self) -> str | None:
             return self.coordinator.data.climate.hvac_mode
         return None
 
-    @property
-    def hvac_modes(self) -> list[str]:
-        """Return the list of available hvac operation modes."""
-        return HVAC_MODES
-
     @property
     def hvac_action(self) -> str | None:
         """Return the current running hvac operation."""
-        is_active = self.coordinator.data.climate.status
-        return CURRENT_HVAC_HEAT if is_active else CURRENT_HVAC_IDLE
-
-    @property
-    def temperature_unit(self) -> str | None:
-        """Return the unit of measurement."""
-        return self.coordinator.data.climate.temp_unit
+        return (
+            CURRENT_HVAC_HEAT
+            if self.coordinator.data.climate.status
+            else CURRENT_HVAC_IDLE
+        )
```

Comment:
This change is not related to the goal of this PR and IMHO don't improve readablity